TISServices/TISServices/Authorization/AuthorizeOpenAMAttribute.cs

```diff
@@ -56,7 +56,7 @@ public override void OnAuthorization(HttpActionContext actionContext)
              {
                  using (HttpClient client = new HttpClient())
                  {
-                     response = client.GetAsync(String.Format("{0}?access_token={1}", OpenAMUrl, authKey)).Result;
+                     response = client.GetAsync(String.Format("{0}?access_token={1}&realm=/sbac", OpenAMUrl, authKey)).Result;

```

Comment:
Not yet. I'm going to double check before and after for that since it does seem like it should be there.  I'm just surprised that we didn't see an issue with it before.  So I'm confused :) . But leave at this point.  Thanks.